os/syscall/syscall_lookup.h

```diff
@@ -216,7 +216,9 @@ SYSCALL_LOOKUP(readdir,                 1, STUB_readdir)
 SYSCALL_LOOKUP(rewinddir,               1, STUB_rewinddir)
 SYSCALL_LOOKUP(seekdir,                 2, STUB_seekdir)
 SYSCALL_LOOKUP(stat,                    2, STUB_stat)
+SYSCALL_LOOKUP(fstat,                   2, STUB_fstat)
 SYSCALL_LOOKUP(statfs,                  2, STUB_statfs)
+SYSCALL_LOOKUP(fstatfs,                 2, STUB_fstatfs)
```

Comment:
There are no STUB_fstatfs, STUB_fstat. add it.

Reply:
I want you to do it in alphabetical order.

Reply:
I will add